app/src/main/java/com/github/sdpteam15/polyevents/view/fragments/MapsFragment.kt

```diff
@@ -128,7 +128,7 @@ class MapsFragment(private val mod: MapsFragmentMod) : Fragment(),
 
                 addNewRouteButton.visibility = View.VISIBLE
                 removeRouteButton.visibility = View.VISIBLE
-                saveNewRouteButton.visibility = View.VISIBLE
+                saveNewRouteButton.visibility = View.INVISIBLE
```

Comment:
You can also do GONE if you don't want it take space

Reply:
Yes but it will be set to visible later in the code